CI/bench/evaluate.py

```diff
@@ -1,142 +1,250 @@
 import argparse
 import os
 import subprocess
+from datetime import datetime
 
 import pandas as pd
 import yaml
 
-parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-
-parser.add_argument(
-    "--config_path", help="path to generated config path to fetch benchmark name", type=str, required=True
-)
-parser.add_argument("--module_name", help="module on which we run benchmark", type=str, required=True)
-parser.add_argument("--time_limit", help="time limit of the benchmark run", type=str, required=True)
-parser.add_argument("--branch_name", help="if it happens to be master then just push the cleaned result, do not evaluate", type=str, required=True)
-
-args = parser.parse_args()
-
-config_path = args.config_path
-module_name = args.module_name
-time_limit = args.time_limit
-branch_name = args.branch_name
-
-for root, dirs, files in os.walk(config_path):
-    for file in files:
-        if file == f"{module_name}_cloud_configs.yaml":
-            config_file = os.path.join(root, file)
-            break
-
-with open(config_file, "r") as f:
-    config = yaml.safe_load(f)
-    benchmark_name = config["benchmark_name"]
-
-subprocess.run(
-    [
-        "agbench",
-        "aggregate-amlb-results",
-        "autogluon-ci-benchmark",
-        module_name,
-        benchmark_name,
-        "--constraint",
-        time_limit,
-    ],
-    check=True,
-)
-
-subprocess.run(
-    [
-        "agbench",
-        "clean-amlb-results",
-        benchmark_name,
-        f"--results-dir-input",
-        f"s3://autogluon-ci-benchmark/aggregated/{module_name}/{benchmark_name}/",
-        "--file-prefix",
-        f"results_automlbenchmark_{time_limit}"
-        "--benchmark-name-in-input-path",
-        "--results-dir-output",
-        "./results",
-    ]
-)
-
-# If it is a PR then perform the evaluation w.r.t cleaned master bench results, test
-if branch_name != "master":
-    paths = []
-    frameworks = []
-    for file in os.listdir("./results"):
-        if file.endswith(".csv"):
-            file = os.path.join("./results", file)
-            df = pd.read_csv(file)
-            paths.append(os.path.basename(file))
-            frameworks += list(df["framework"].unique())
-
-    modified_list_paths = []
-    modified_list_frameworks = []
-
-    for path in paths:
-        modified_list_paths.append('--paths')
-        modified_list_paths.append(path)
-
-    for framework in frameworks:
-        modified_list_frameworks.append('--frameworks-run')
-        modified_list_frameworks.append(framework)
+def process_results(eval_flag: bool):
+    try:
+        paths = []
+        frameworks = []
+        for file in os.listdir("./results"):
+            if file.endswith(".csv") and not file.endswith("_min.csv") and not file.endswith(".parquet"):
```

Comment:
I think `not file.endswith(".parquet")` is not necessary here

Reply:
Adding this for robustness in case evaluation scripts are changed and more files are added.

Reply:
anything `.endswith(".csv")` will not end with `.parquet`, so the last part is redundant.